pkgs/development/tools/misc/src-cli/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "src-cli";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "sourcegraph";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-cd96ddFtPOMATI0MEMFH399/y7SZeASetVhvRt0P38A=";
+  };
+
+  vendorSha256 = "sha256-ll/Q7hZJngOGG/Jl79KvEIHQ+ARDxFD7KHY3nSokRi0=";
+
+  subPackages = [ "cmd/src" ];
+
+  CGO_ENABLED = 0;
```

If i recall correctly this has the same result, but looks a bit nicer
```suggestion
  CGO_ENABLED = false;
```